laborator/lab-12/01-one-tut/b-static/Makefile

```diff
@@ -0,0 +1,18 @@
+CFLAGS = -fno-PIC -m32 -g
+LDFLAGS = -static -no-pie -m32
+
+.DEFAULT_GOAL: all
+
+.PHONY: all clean
+
+all: name
```

Comment:
Sa modifici name in hello in makefile-ul asta

Reply:
Done.